nekoyume/Assets/_Scripts/UI/Widget/Popup/NoticePopup.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Globalization;
+using UnityEngine.UI;
+using UnityEngine;
+using Nekoyume.Game.Controller;
+
+namespace Nekoyume.UI
+{
+    public class NoticePopup : PopupWidget
+    {
+        [SerializeField] private Button detailButton;
+        [SerializeField] private Button closeButton;
+        private const string LastNoticeDayKeyFormat = "NOTICE_POPUP_LAST_DAY_{0}";
+        
+        // Notice Info
+        private const string NoticeName = "ItemLevelRequirement";
+        private const string NoticeBeginTime = "2022/03/01 00:00:00";
+        private const string NoticeEndTime = "2022/03/31 00:00:00";
+        private const string NoticePageUrlFormat = "https://www.notion.so/planetarium/1bc6de399b3b4ace95fca3a3020b4d79";
+
+        private static bool CanShowNoticePopup
+        {
+            get
+            {
+                var worldInfo = Game.Game.instance.States.CurrentAvatarState.worldInformation;
+                if (worldInfo is null) return false;
+                var clearedStageId = worldInfo.TryGetLastClearedStageId(out var id) ? id : 1;
+                if (TutorialController.GetCheckPoint(clearedStageId) != 0) return false;
+                
+                var tutorialControllerIsPlaying = Game.Game.instance.Stage.TutorialController.IsPlaying;
+                if (tutorialControllerIsPlaying) return false;
```

Comment:
I think it can be replaced with `TutorialController.IsPlaying`

Reply:
TutorialController or IsPlaying is not static. so It must call singleton's property.